ViViaN/vivianInstall.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+usage()
+{
+    cat << EOF
+    usage: $0 options
+
+    OPTIONS:
+      -h    Show this message
+      -s    Script Directory
+      -i    Instance name (Namespace/Database for Caché)
+
+EOF
+}
+
+while getopts ":hi:s" option
+do
+    case $option in
+        h)
+            usage
+            exit 1
+            ;;
+        s)
+            scriptdir=$OPTARG
+            ;;
+        i)
+            instance=$(echo $OPTARG |tr '[:upper:]' '[:lower:]')
+            ;;
+    esac
+done
+
+timestamp() {
+    date +"%Y-%m-%d-%H:%M:%S"
+}
+
+if [[ -z $instance ]]; then
+    instance=osehra
+fi
+
+if [[ -z $scriptdir ]]; then
+    scriptdir=/opt/vista
+fi
+
+yum install -y httpd graphviz java-1.8.0-openjdk-devel php
+curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" && \
+    python get-pip.py && \
+    pip install xlrd reportlab
+# cp $scriptdir/ViViaN/viv.conf /etc/httpd/conf.d
+basedir=/opt/cachesys/$instance
+
+# Add apache to start.sh
+awk -v n=5 -v s='echo "Starting Apache"' 'NR == n {print s} {print}' $basedir/bin/start.sh > $basedir/bin/start.tmp && mv $basedir/bin/start.tmp $basedir/bin/start.sh
+awk -v n=6 -v s="/usr/sbin/apachectl" 'NR == n {print s} {print}' $basedir/bin/start.sh > $basedir/bin/start.tmp && mv $basedir/bin/start.tmp $basedir/bin/start.sh
+
+# Fix start.sh permissions
+chown cacheusr$instance:cachegrp$instance $basedir/bin/start.sh
+chmod +x $basedir/bin/start.sh
+
+sh $basedir/bin/start.sh &
+mkdir -p /opt/VistA-docs
+mkdir -p /opt/viv-out
+pushd /opt
+echo "Downloading OSEHRA VistA"
+curl -fsSL --progress-bar https://foia-vista.osehra.org/VistA_Integration_Agreement/2018_January_22_IA_Listing_Descriptions.TXT -o ICRDescription.txt
+#change from default to test capitalization changes
+curl -fsSL --progress-bar https://github.com/OSEHRA/VistA/archive/master.zip -o VistA-master.zip
```

Comment:
I'd put the echo statement above here. However this is all the support libraries for VistA - not the actual VistA files.